backend/src/service/board.ts

```diff
@@ -0,0 +1,15 @@
+import { BoardRepository, boardRepository } from '../repository/board';
+import { DTO } from '../Dto';
+
+class BoardService {
+  constructor(private boardRepo: BoardRepository) {}
+
+  // Update task status
+
+  async updateStatus(ID: DTO.ID, body: DTO.ITask): Promise<DTO.ITaskDoc | null> {
```

Comment:
it is not clear `ID` of which entity it is used here and why is it uppercase?

Reply:
Seems like you are not updating the board status but task status, then why is it in the board service?